src/tools/cephfs/top/cephfs-top

```diff
@@ -786,6 +796,20 @@ class FSTop(FSTopBase):
                         wrap(client_meta[CLIENT_METADATA_MOUNT_ROOT_KEY], hlen), curses.A_DIM)
                 else:
                     self.fsstats.addstr(y_coord, xp, "N/A", curses.A_DIM)
+            elif item == FS_TOP_MAIN_WINDOW_COL_MNTPT_TYPE:
```

Comment:
update `hlen` statement is needed only once in this else case.

Reply:
Addressed